template/README.md

```diff
@@ -15,10 +15,10 @@ $ composer require {org}/{repo}
 
 ## Documentation
 
-Documentation is [in the doc tree](docs/book/), and can be compiled using [mkdocs](http://www.mkdocs.org):
+Browse the documentation online at https://docs.zendframework.com/{repo}/
 
-```bash
-$ mkdocs build
-```
+## Support
 
-You may also [browse the documentation online](https://docs.zendframework.com/{repo}/).
+* [Issues](https://github.com/zendframework/{repo}/issues/)
+* [Chat](https://zendframework-slack.herokuapp.com/)
```

Comment:
Everywhere we are using "Slack" not "Chat" so maybe we should change it here as well?

Reply:
I prefer the word "Chat", because if we use "Slack" then we must also use "Discourse" for "Forum" etc.
But who forces us to use company names, rather than the names which are intelligible to all?

Reply:
If we use "Chat" then probably we should update it everywhere, for example:
https://github.com/zendframework/maintainers/blob/master/template/docs/SUPPORT.md
and composer.json, and documentation ... etc.

Reply:
@webimpress 
Not yet, because at the moment it is still a suggestion. The same suggestion is include for documentation theme: https://github.com/zendframework/zf-mkdoc-theme/pull/34 (see homepage screenshot)

Reply:
I like "chat" here. If we change it later, the verbiage stays the same.